### PR TITLE
Flappy Bird

### PR DESCRIPTION
**Pull request by Yaser-123**
Flappy Bird is an addictive and challenging web-based game where players control a bird attempting to fly between rows of green pipes without hitting them. The game is built using HTML, CSS, and JavaScript, offering a simple yet highly engaging experience.

### CODE REVIEW

**Comment by Yaser-123** [comment]
https://github.com/dhairyagothi/50_days_50_web_project/assets/140265777/473860a2-2ebe-4fc5-a3b2-a6acd442f884

**Comment by dhairyagothi** [comment]
image

don't use unnecessary spaces 

image

resolve it here only @Yaser-123

File: public/index.html [hunk 1]
```diff
@@ -9,37 +9,167 @@
...
+        <div class="table">
+            <div class="day-number">Day 29</div>
+            <div class="project-name">Snake Game</div>
+            <a href="./snake_game/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 30</div>
+            <div class="project-name">Flappy-bird-game</div>
+            <a href="./Flappy-bird-main/index.html" class="demo-link">Demo</a>
+        </div>
```
Comment (dhairyagothi) [inline public/index.html:167]: instead of adding and deleting the content of a file 
PULL the PR
this is not a good practise

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by Yaser-123** [comment]
Removed Unwanted spaces

**Comment by dhairyagothi** [comment]
good to go now @Yaser-123

**Review by dhairyagothi — APPROVED** [review]

**Pull request merged**